mypy/test/data.py

```diff
@@ -137,19 +142,29 @@ def add_dirs(self, dir: str) -> List[str]:
             os.mkdir(dir)
             return dirs
 
-    def run(self):
-        if self.name.endswith('-skip'):
-            raise SkipTestCaseException()
+    def run(self, obj: Any = None):
+        if obj is None:
```

Comment:
What is this `obj` parameter? For understanding this logic it'd be really helpful to have a description of it somewhere.

Perhaps even a type. :-)